pusher/src/Model/PusherRoom.ts

```diff
@@ -55,4 +56,13 @@ export class PusherRoom {
     public isEmpty(): boolean {
         return this.positionNotifier.isEmpty();
     }
+
+    public isUpToDate(versionNumber: number): boolean {
```

Comment:
The name of the method is misleading.

"isser" should not mutate the object. This method is both reading and writing state.

Maybe we could rename it to `updateVersion` ?